mdx-web/src/main/java/com/mx/path/model/mdx/web/controller/PaymentsController.java

```diff
@@ -17,6 +19,30 @@
 @RequestMapping(value = "{clientId}/users/{user_id}", produces = BaseController.MDX_MEDIA)
 public class PaymentsController extends BaseController {
 
+  @RequestMapping(value = "/payments/enrollment", method = RequestMethod.GET, consumes = BaseController.MDX_MEDIA)
+  public final ResponseEntity<Enrollment> getPaymentEnrollStatus() {
+    AccessorResponse<Enrollment> response = gateway().payments().enrollmentStatus();
+    return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), HttpStatus.OK);
+  }
+
+  @RequestMapping(value = "/payments/enrollment", method = RequestMethod.PUT, consumes = BaseController.MDX_MEDIA)
+  public final ResponseEntity<Enrollment> setPaymentEnrollStatus(@RequestBody Enrollment enrollmentRequest) {
+    AccessorResponse<Enrollment> response = gateway().payments().updateEnrollmentStatus(enrollmentRequest);
+    return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), HttpStatus.OK);
+  }
+
+  @RequestMapping(value = "/payments/settings", method = RequestMethod.GET, consumes = BaseController.MDX_MEDIA)
+  public final ResponseEntity<Settings> getPaymentSettings() {
+    AccessorResponse<Settings> response = gateway().payments().settings();
+    return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), HttpStatus.OK);
+  }
+
+  @RequestMapping(value = "/payments/settings", method = RequestMethod.PUT, consumes = BaseController.MDX_MEDIA)
+  public final ResponseEntity<Settings> setPaymentSettings(Settings settingsRequest) {
```

Comment:
This should be `@RequestBody Settings settingsRequest`.

Reply:
Updated